.github/ISSUE_TEMPLATE/documentation_request.yml

```diff
@@ -0,0 +1,53 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Documentation Request
+description: Request updates or additions to cuDF's documentation
+title: "[DOC] "
+labels: ["doc"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to help cuDF and improve our documentation.
+        - By submitting this issue, you agree to follow our [Code of Conduct](https://docs.rapids.ai/resources/conduct/)
+        - Please link to any related [open documentation issues](https://github.com/rapidsai/cudf/issues?q=is%3Aopen+is%3Aissue+label%3Adoc)
+
+  - type: dropdown
+    id: new_or_correction
+    attributes:
+      label: Is this for new documentation, or an update to existing docs?
+      options:
+        - New
+        - Update
+
+  - type: textarea
+    id: problem
+    attributes:
+      label: Describe the incorrect/missing documentation.
+      placeholder: "Example: A code snippet mentions function foo(args) but I cannot find any documentation on it."
+    validations:
+      required: true
+
+  - type: textarea
+    id: search_locs
+    attributes:
+      label: |
+        If this is a correction, please provide a link to the incorrect documentation.
+        If this is a new documentation request, please link to where you have looked.
+      placeholder: |
```

Comment:
This placeholder isn't helpful because you can't copy the link. Can you link this in the label field and remove the placeholder?